.github/workflows/maven.yml

```diff
@@ -34,3 +32,33 @@ jobs:
         run: ./mvnw -B clean install
       - name: code coverage
         run: bash <(curl -s https://codecov.io/bash)
+
+      - if: github.ref == 'refs/heads/develop'
+        name: Set S3 folder name and quay tag name
+        run: |
+          S3_FOLDER=${GITHUB_REF##refs/tags/}
+          if [ $GITHUB_REF == $S3_FOLDER ]; then
+           # If this isn't a tag, it must be a branch
+           S3_FOLDER=${GITHUB_REF##refs/heads/}
+          fi
+          # QUAY_TAG differs from S3_FOLDER because it only contains the git ref with no short hash. It also replaces slashes with underscores.
+          echo "QUAY_TAG=${S3_FOLDER//\//_}" >> $GITHUB_ENV
+          echo "S3_FOLDER=${S3_FOLDER}-$(echo $GITHUB_SHA | cut -c -7)" >> $GITHUB_ENV
+          
+      # neat, quay itself uses manual github actions https://github.com/quay/quay/blob/master/.github/workflows/build-and-publish.yaml
+      - if: github.ref == 'refs/heads/develop'
```

Comment:
At some point in the future, especially if we end up creating additional Docker images, we might want to consider moving this code to dockstore/workflows. But looks good now.

Reply:
Agreed, I noticed we create images and upload checksums for images in dockstore/dockstore and dockstore-deploy as well, so it'll be good to create a re-usable workflow. I'll create a ticket for this or maybe this can be tackled when we implement uploading the image digest for this image